bmgui/parse_corpus_page.py

```diff
@@ -37,12 +37,18 @@
 
 
 class CheckBoxCorpuses:
+    """Displays status of parsing files through Checkbutton tkinter widgets.
 
-    """
-    Petit automat permettant d'afficher sur la même ligne :
-        - L'annee du corpus
-        - Wos rawdata/parsing dispo
-        - Scopus rawdata/parsing dispo
+    Args:
+        year (str): Corpus year defined by 4 digits.
+        wos_r (bool): Status of WoS raw-data file.
+        wos_p (bool): Status of WoS parsing files.
+        scopus_r (bool): Status of Scopus raw-data file.
+        scopus_p (bool): Status of Scopus parsing files.
+        concat (bool) : Status of concatenation and deduplication files.
+
+    Returns:
+        None.
```

Comment:
A class does not return anything (must be removed)

Reply:
Taken into account. Thanks